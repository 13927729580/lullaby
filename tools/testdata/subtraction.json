{
  "components": [{
    "def_type": "~RenderDef",
    "def": {
    }
  }, {
    "def_type": "~AnimationResponseDef",
    "def": {
    }
  }]
}
