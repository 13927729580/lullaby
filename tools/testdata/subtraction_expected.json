{
  "components": [{
    "def_type": "TransformDef",
    "def": {
    }
  }, {
    "def_type": "ReticleDef",
    "def": {
      "thickness": 0.1,
      "diameter": 0.3
    }
  }]
}
